Assignment6.Rmd

```diff
@@ -66,18 +76,32 @@ M2$predict1 <- predict(c.tree1, M2, type = "class")
 
 M2$predict2 <- predict(c.tree2, M2, type = "class")
 
-table(M2$certified, M2$predict1)
+T1 <- table(M2$certified, M2$predict1)
+
+T2 <- table(M2$certified, M2$predict2)
 
-table(M2$certified, M2$predict2)
+#Tree 1 True Results
+T1TR <- (T1[2,2]+T1[1,1])/sum(T1)
+
+#Tree 2 True Results
+T2TR <- (T2[2,2]+T2[1,1])/sum(T2)
 
 ```
+#Comment
+The second tree has a lower error rate. This is because the True Negative and True Positive for tree 2 is 53.63% while tree 1 is 21.86% so that means the second tree has a higher accuracy vesus the first tree.
```

Comment:
What else does this suggest about the second tree in terms of its generalizability?